### ISSUE TITLE
Base Example Breaks

### ISSUE THREAD

**Issue by koaning** [danielfrg/mkdocs-jupyter#3]
When running a base example with this notebook; 

![image](https://user-images.githubusercontent.com/1019791/78469637-675e1080-7723-11ea-944e-eae9a00f6a98.png)

Then running `mkdocs serve` gives a hard to parse error. This is the trace: 

```
(venv) ➜  scikit-fairness git:(documentation) ✗ mkdocs serve
INFO    -  Building documentation... 
INFO    -  Cleaning site directory 
INFO    -  The following pages exist in the docs directory, but are not included in the "nav" configuration:
  - fairness.ipynb 
Converting: /Users/vincent/Development/scikit-fairness/docs/fairness.ipynb
ERROR   -  Error reading page 'fairness.ipynb': string indices must be integers 
Traceback (most recent call last):
  File "/Users/vincent/Development/scikit-fairness/venv/bin/mkdocs", line 8, in <module>
    sys.exit(cli())
  File "/Users/vincent/Development/scikit-fairness/venv/lib/python3.7/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/Users/vincent/Development/scikit-fairness/venv/lib/python3.7/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/Users/vincent/Development/scikit-fairness/venv/lib/python3.7/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Users/vincent/Development/scikit-fairness/venv/lib/python3.7/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/vincent/Development/scikit-fairness/venv/lib/python3.7/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/Users/vincent/Development/scikit-fairness/venv/lib/python3.7/site-packages/mkdocs/__main__.py", line 143, in serve_command
    **kwargs
  File "/Users/vincent/Development/scikit-fairness/venv/lib/python3.7/site-packages/mkdocs/commands/serve.py", line 141, in serve
    config = builder()
  File "/Users/vincent/Development/scikit-fairness/venv/lib/python3.7/site-packages/mkdocs/commands/serve.py", line 136, in builder
    build(config, live_server=live_server, dirty=dirty)
  File "/Users/vincent/Development/scikit-fairness/venv/lib/python3.7/site-packages/mkdocs/commands/build.py", line 274, in build
    _populate_page(file.page, config, files, dirty)
  File "/Users/vincent/Development/scikit-fairness/venv/lib/python3.7/site-packages/mkdocs/commands/build.py", line 174, in _populate_page
    page.render(config, files)
  File "/Users/vincent/Development/scikit-fairness/venv/lib/python3.7/site-packages/mkdocs_jupyter/plugin.py", line 57, in new_render
    self.toc = get_nb_toc(page.file.abs_src_path)
  File "/Users/vincent/Development/scikit-fairness/venv/lib/python3.7/site-packages/mkdocs_jupyter/plugin.py", line 74, in get_nb_toc
    toc = get_toc(getattr(md, "toc", ""))
  File "/Users/vincent/Development/scikit-fairness/venv/lib/python3.7/site-packages/mkdocs/structure/toc.py", line 11, in get_toc
    toc = [_parse_toc_token(i) for i in toc_tokens]
  File "/Users/vincent/Development/scikit-fairness/venv/lib/python3.7/site-packages/mkdocs/structure/toc.py", line 11, in <listcomp>
    toc = [_parse_toc_token(i) for i in toc_tokens]
  File "/Users/vincent/Development/scikit-fairness/venv/lib/python3.7/site-packages/mkdocs/structure/toc.py", line 59, in _parse_toc_token
    anchor = AnchorLink(token['name'], token['id'], token['level'])
TypeError: string indices must be integers
```

**Comment by koaning** [comment]
This error also occurs when `toc` is not a markdown extension and it also occurs when the notebook is not listed in the `nav` to be rendered.

**Comment by koaning** [comment]
It seems to run fine with mkdocs version 1.0.4. But that version is incompatible with `mkdocstrings`.

**Comment by koaning** [comment]
To others who may be looking for a solution, there is an alternative plugin called `mknotebooks`. I prefer the style of this package though so I am hoping to help the maintainer fix it.

**Comment by danielfrg** [comment]
Thanks for the reports and the Pytest PR!
I am interested on keeping this alive so I will take a look when I have the chance!

**Comment by koaning** [comment]
Sweet. Let me know if you require clarity.

**Comment by MasterScrat** [comment]
> This error also occurs when `toc` is not a markdown extension and it also occurs when the notebook is not listed in the `nav` to be rendered.

I addressed both these points and it still didn't work for me.

**Comment by ministp** [comment]
i have a same question

**Comment by danielfrg** [comment]
Hi all, this is fixed now and released on pypi: https://pypi.org/project/mkdocs-jupyter/0.11.0/
